xml/cl.xml

```diff
@@ -5622,7 +5622,7 @@ server's OpenCL/api-docs repository.
                 <command name="clIcdGetPlatformIDsKHR"/>
             </require>
         </extension>
-        <extension name="cl_loader_layers" supported="opencl">
+        <extension name="cl_loader_layers" revision="1.0.0" supported="opencl">
```

Comment:
I've set this to `1.0.0` but can find the spec for it

Reply:
I think the latest spec is here: https://github.com/KhronosGroup/OpenCL-Docs/blob/main/extensions/cl_loader_layers.asciidoc

Looks like version `1.0.0` should be fine.

@Kerilk should we build an HTML version of this spec and post it on the registry?  We published an HTML spec for [cl_loader_info](https://registry.khronos.org/OpenCL/extensions/loader/cl_loader_info.html), but it looks like we haven't published cl_loader_layers.